.clang-format

```
@@ -1,4 +1,5 @@
 Language: Cpp
 BasedOnStyle: Google
 IndentWidth: 2
-ColumnLimit: 0
\ No newline at end of file
+ColumnLimit: 0
+IncludeBlocks: Preserve
```

Add newline at end of file

```suggestion
IncludeBlocks: Preserve

```